pkg/asset/installconfig/powervs/validation.go

```diff
@@ -28,6 +28,15 @@ func Validate(config *types.InstallConfig) error {
 			fldPath := field.NewPath("compute").Index(idx)
 			allErrs = append(allErrs, validateMachinePool(fldPath, &compute)...)
 		}
+		// Machine pool CIDR check
+		for i := range config.Networking.MachineNetwork {
+			// Each machine pool CIDR must have 24 significant bits (/24)
+			if !(strings.HasSuffix((&config.Networking.MachineNetwork[i].CIDR).String(), "/24")) {
```

Comment:
What about
```suggestion
			if bits, _ := config.Networking.MachineNetwork[i].CIDR.Mask.Size(); bits != 24 {
```

Reply:
Good point @r4f4 that looks much cleaner